> Because GitLab CI failed (at least on paraview, with no logs (too much output) and may be more) I rebased to relatively recent develop to let GitLab CI give this another run over the weekend.

Same issue with paraview:
```
error: '_FillValue' undeclared (first use in this function); did you mean 'NC_FillValue'?
```

<details><summary>(full failing line)</summary>
</pre>
[2309/12590] /home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeh/linux-x86_64_v3/compiler-wrapper-1.0-rwnqblirwn5aenlmsbvsieo7mzw672ee/libexec/spack/gcc/gcc -DMPICH_SKIP_MPICXX -DMPI_NO_CPPBIND -DOMPI_SKIP_MPICXX -DVTK_MODULE_ENABLE_VTK_mpi=1 -D_MPICC_H -DexoIIc_EXPORTS -DexodusII_EXPORTS -I/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII/vtkexodusII -I/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/exodusII/vtkexodusII -I/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII/vtkexodusII/include -I/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/exodusII/vtkexodusII/include -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/exodusII -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/hdf5 -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/hdf5 -isystem /home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeh/linux-x86_64_v3/hdf5-1.14.6-he5dt47v3wxznexhx6bmnbg4ksjbdrx2/include -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/Utilities/MPI -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/Utilities/MPI -isystem /home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeh/linux-x86_64_v3/mpich-4.3.1-u5fz7x52imd6ighhgjyeyrhkp7edm262/include -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/netcdf -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/netcdf -isystem /home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeh/linux-x86_64_v3/netcdf-c-4.9.3-hbzpxv426sflitezwt3lvsxr27bn4jzy/include -O3 -DNDEBUG -std=gnu99 -fPIC -MD -MT VTK/ThirdParty/exodusII/vtkexodusII/CMakeFiles/exodusII.dir/src/ex_put_prop_names.c.o -MF VTK/ThirdParty/exodusII/vtkexodusII/CMakeFiles/exodusII.dir/src/ex_put_prop_names.c.o.d -o VTK/ThirdParty/exodusII/vtkexodusII/CMakeFiles/exodusII.dir/src/ex_put_prop_names.c.o -c /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII/vtkexodusII/src/ex_put_prop_names.c
FAILED: [code=1] VTK/ThirdParty/exodusII/vtkexodusII/CMakeFiles/exodusII.dir/src/ex_put_prop_names.c.o 
/home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeh/linux-x86_64_v3/compiler-wrapper-1.0-rwnqblirwn5aenlmsbvsieo7mzw672ee/libexec/spack/gcc/gcc -DMPICH_SKIP_MPICXX -DMPI_NO_CPPBIND -DOMPI_SKIP_MPICXX -DVTK_MODULE_ENABLE_VTK_mpi=1 -D_MPICC_H -DexoIIc_EXPORTS -DexodusII_EXPORTS -I/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII/vtkexodusII -I/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/exodusII/vtkexodusII -I/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII/vtkexodusII/include -I/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/exodusII/vtkexodusII/include -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/exodusII -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/hdf5 -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/hdf5 -isystem /home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeh/linux-x86_64_v3/hdf5-1.14.6-he5dt47v3wxznexhx6bmnbg4ksjbdrx2/include -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/Utilities/MPI -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/Utilities/MPI -isystem /home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeh/linux-x86_64_v3/mpich-4.3.1-u5fz7x52imd6ighhgjyeyrhkp7edm262/include -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-build-wmb52tm/VTK/ThirdParty/netcdf -isystem /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/netcdf -isystem /home/software/spack/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeholder__/__spack_path_placeh/linux-x86_64_v3/netcdf-c-4.9.3-hbzpxv426sflitezwt3lvsxr27bn4jzy/include -O3 -DNDEBUG -std=gnu99 -fPIC -MD -MT VTK/ThirdParty/exodusII/vtkexodusII/CMakeFiles/exodusII.dir/src/ex_put_prop_names.c.o -MF VTK/ThirdParty/exodusII/vtkexodusII/CMakeFiles/exodusII.dir/src/ex_put_prop_names.c.o.d -o VTK/ThirdParty/exodusII/vtkexodusII/CMakeFiles/exodusII.dir/src/ex_put_prop_names.c.o -c /tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII/vtkexodusII/src/ex_put_prop_names.c
/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII/vtkexodusII/src/ex_put_prop_names.c: In function 'vtkexodusII_ex_put_prop_names':
/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII/vtkexodusII/src/ex_put_prop_names.c:175:54: error: '_FillValue' undeclared (first use in this function); did you mean 'NC_FillValue'?
  175 |     if ((status = nc_put_att_longlong(exoid, propid, _FillValue, int_type, 1, vals)) != NC_NOERR) {
      |                                                      ^~~~~~~~~~
      |                                                      NC_FillValue
/tmp/root/spack-stage/spack-stage-paraview-5.13.3-wmb52tmzatlxxgvrjdmlnr75kwv3y7eq/spack-src/VTK/ThirdParty/exodusII/vtkexodusII/src/ex_put_prop_names.c:175:54: note: each undeclared identifier is reported only once for each function it appears in
</pre>
</details>

--- a/repos/spack_repo/builtin/packages/paraview/package.py
+++ b/repos/spack_repo/builtin/packages/paraview/package.py
@@ -288,6 +288,7 @@ class Paraview(CMakePackage, CudaPackage, ROCmPackage):
     depends_on("libtheora")
     depends_on("libtiff")
     depends_on("netcdf-c")
+    depends_on("netcdf-c@:4.9.2", when="@:5.13")
     depends_on("pegtl@2.8.3")
     depends_on("protobuf@3.4:")
     # Paraview 5.10 can't build with protobuf > 3.18

